publisher/snippets/java/V2APIMigration.java

```diff
@@ -26,15 +26,15 @@ public static void main(String[] args) throws Exception {
     }
 
     public static void ExampleTokenGeneration(String apiKey, String secretKey) throws Exception {
-        // documentation: https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/endpoints/post-token-generate.md
+        // documentation: https://unifiedid.com/docs/endpoints/post-token-generat

```

Comment:
// documentation: https://unifiedid.com/docs/endpoints/post-token-generate
(it's missing the last letter)